content/en/Platform Deep Dive/Organization/automated-data-removal.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Automated Data Removal"
+linkTitle: "Automated Data Removal"
+weight: 40
+description: >
+  Process of automatically deleting data from the Cobalt Platform based on predefined rules and schedules.
+---
+
+{{% pageinfo %}}
+When your organization's contract expires, the associated data will be removed after a predefined period of time. This period can be extended based on certain conditions. Learn more about our [privacy policy](https://cobalt.io/terms#privacy).
+{{% /pageinfo %}}
+
+When your contract ends, a 15-day grace period begins during which you will still have access to the platform to [export any necessary data](/platform-deep-dive/organization/your-contract/#download-your-organizations-data). Organization Owners will also receive a notification about this. If the contract is renewed, the grace period will be lifted.
+
+{{% image src="/deepdive/ContractGateAfterGracePeriod.png" alt="contract gate after grace period" class="mt0" %}}
+
+After the grace period, you will still be able to sign in to the platform but will no longer have access to your organization's data. However, this data will remain in our system for 90 days after the contract ends (termination period).
+
+{{% image src="/deepdive/ConfirmToRetainData.png" alt="confirm to retain data" class="mt0" %}}
+
+You can extend this termination period by an additional 90 days if confirmed before the end of the initial period. The termination period can only be extended once and only organization owners will be able to do this.
+If you wish to have your data removed earlier, please reach out to your Customer Success Manager (CSM).
+
+## Data Subject to Removal
+
+The following data will be removed:
+
+- User accounts (except those associated with additional organizations not affected by data removal)
+- Organization's address, name, and logo
+- Assets
+- Pentests
+- Jira integrations
+
+## Notification Recipients
+
+The following user roles will receive email notifications about data removal:
+
+- Organization Owners
+- Customer Success Managers
+
+## Notification Schedule
+
+{{% image src="/deepdive/DataRemovalReminder.png" alt="data removal reminder" class="mt0" %}}
```

Comment:
Just realized - maybe it's better to show an image of the email reminder that shows the Keep My Data option since that's the version users will see first

<img width="488" alt="InitialEmailReminder" src="https://github.com/user-attachments/assets/36371b9d-6f73-40a6-bf66-5b1cb8f740b3" />